astronomer/providers/google/cloud/operators/dataproc.py

```diff
@@ -190,22 +190,29 @@ def execute(self, context: Context) -> None:
             metadata=self.metadata,
         )
 
-        end_time: float = time.time() + self.timeout
+        try:
+            hook.get_cluster(project_id=self.project_id, region=self.region, cluster_name=self.cluster_name)
+        except NotFound:
+            return
```

Comment:
We should log a message/warning here that the cluster was not found.

Reply:
Just added it. Thanks!